### PR TITLE
Update and fix `highest-rated-comment`

### LINKED ISSUE #6196: `highest-rated-comment` has no comment icon and 3px chevron
### Description

1. The green chevron border for `highest-rated-comment` is 2px too wide (can remove that CSS rule to fix it).
2. The logic to add the `highest-rated-comment` header icon uses the wrong CSS class.

https://github.com/refined-github/refined-github/blob/3f2ebe8/source/features/highest-rated-comment.css#L10-L12

https://github.com/refined-github/refined-github/blob/3f2ebe8/source/features/highest-rated-comment.tsx#L54

```
❌ highest-rated-comment
refined-github.js:3667 📕 22.11.21 → TypeError: Cannot read properties of undefined (reading 'before')
    at refined-github.js:8074:88
    at init (refined-github.js:8078:10)
    at runFeature (refined-github.js:3709:26)
    at setupPageLoad (refined-github.js:3715:40)
    at refined-github.js:3736:35
```

### How to replicate the issue + URL

image

https://github.com//ytdl-org/youtube-dl/issues/30568#issuecomment-1024981915

### Extension version

22.11.21

### Browser(s) used

Edge 108

### PR DESCRIPTION
**Pull request by meche-gh**
Closes #6196.

## Test URLs
https://github.com//ytdl-org/youtube-dl/issues/30568#issuecomment-1024989514

## Screenshot
image
image

### CODE REVIEW

File: source/features/highest-rated-comment.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ function getBestComment(): HTMLElement | undefined {
 
 function highlightBestComment(bestComment: Element): void {
 	select('.unminimized-comment', bestComment)!.classList.add('rgh-highest-rated-comment');
-	select('.unminimized-comment .timeline-comment-header-text', bestComment)!.before(
+	select('.unminimized-comment .text-normal', bestComment)!.before(
```
Comment (fregante) [inline source/features/highest-rated-comment.tsx:54]: This selector is too generic now, it could appear anywhere

**Review by fregante — COMMENTED** [review]
Thank you for the PR!

File: source/features/highest-rated-comment.tsx (hunk 1 above)
Comment (meche-gh) [reply source/features/highest-rated-comment.tsx:54]: Can fix 🙂
```suggestion
	select('.unminimized-comment .timeline-comment-header h3', bestComment)!.before(
```

Comment (meche-gh) [reply source/features/highest-rated-comment.tsx]: Resolved by a7c9479...477c01e

**Pull request merged**